src/crate/crash/command.py

```diff
@@ -494,9 +503,37 @@ def main():
 
     crate_hosts = [host_and_port(h) for h in args.hosts]
     error_trace = args.verbose > 0
+
+    force_passwd_prompt = args.password
+    password = None
+
+    # If password prompt is not forced try to get it from env. variable.
+    if not force_passwd_prompt:
+        password = os.environ.get('CRATEPWD', '')
```

Comment:
password should stay `None` if CRATEPWD is not available

Reply:
in my opinion, PWD is not a common abbreviation for password.
either PW or PASSWD